docs/site-generator/hugo-template/Makefile

```diff
@@ -0,0 +1,78 @@
+# Default shell
+SHELL := /bin/bash
+
+# Config
+DESTINATION ?= public
+LANDING_PATH ?=
+
+# System
+url_os = 64bit
```

Comment:
who installs hugo and how is the right hugo version being controlled?
I think the right way is controlling it from the `site-generator` since it is the responsible of building the landing-like site and gathering the right templates with the right hugo sintax.